symplyphysics/laws/relativistic/total_energy_via_relativistic_mass.py

```diff
@@ -0,0 +1,23 @@
+from sympy import Eq, solve
+from symplyphysics import units, Quantity, Symbol, validate_input, validate_output, symbols
+
+# Description
+## Fundamentally the energy of an object is synonimical to its mass.
+
+# Law: E = m * c**2, where
+## E is total (or relativistic) energy of body/system,
+## m is mass (relativistic mass, not only rest mass),
+## c is speed of light.
+
+energy = Symbol("energy", units.energy)
+mass = symbols.basic.mass
```

Comment:
Maybe it's better to use other name? It's easy to mistake it for rest mass.

Reply:
Makes sense